README.md

```diff
@@ -15,3 +15,20 @@ Please refer to the
 - [Intel Edison] (./INTEL/EDISON)
 - [Ti CC3200] (./TI/CC3200)
 - [Linux] (./Linux)
+
+## Create tasks
+
+This SDK requests creating tasks by KII\_TASK\_CREATE callback.  
+
+When you need to set stack size yourself, please check task's stack size.
```

Comment:
You'll specify stack size, depth and priority, etc depending on the environment.
Please check samples for reference.